keps/0004-add-testing-infrastructure.md

```diff
@@ -145,8 +179,7 @@ How will we know that this has succeeded?
 
 - When repository owners can feel confident that code changes are not breaking functionality.
 - Tests pass for the API objects
-- Leverage testing scaffolding provided by (and subsequently removed by us) Kubebuilder for
-  controller logic.
+- Leverage testing scaffolding provided by (and subsequently removed by us) Kubebuilder for controller logic.
```

Comment:
I don't understand this point 🤔

Reply:
Replaced it with something more relevant